x-pack/plugins/security_solution/public/timelines/components/timeline/body/column_headers/header/helpers.ts

```diff
@@ -35,7 +35,7 @@ export const getNextSortDirection = (currentSort: Sort): Direction => {
     case 'none':
       return Direction.desc;
     default:
-      return assertUnreachable(currentSort.sortDirection, 'Unhandled sort direction');
+      return assertUnreachable(currentSort.sortDirection as never, 'Unhandled sort direction');
```

Comment:
via `x-pack/plugins/security_solution/common/utility_types.ts`:

```
 * If you see an error, DO NOT cast "as never" such as:
 * assertUnreachable(x as never) // BUG IN YOUR CODE NOW AND IT WILL THROW DURING RUNTIME
 * If you see code like that remove it, as that deactivates the intent of this utility.
 * If you need to do that, then you should remove assertUnreachable from your code and
 * use a default at the end of the switch instead.
```

Reply:
Make sense!!!